zuul-core/src/main/java/com/netflix/zuul/origins/BasicNettyOriginManager.java

```diff
@@ -52,3 +52,4 @@ public BasicNettyOrigin createOrigin(String name, String vip, String uri, boolea
         return new BasicNettyOrigin(name, vip, registry);
     }
 }
+class foo {}
```

Comment:
bar?

Reply:
Whoops, I guess that's why I ask for code reviews. :P